src/OrchardCore/OrchardCore.Media.Core/GraphQL/MediaFieldQueryObjectType.cs

```diff
@@ -6,7 +6,7 @@
 using OrchardCore.Apis.GraphQL;
 using OrchardCore.Media.Fields;
 
-namespace OrchardCore.Media.GraphQL
+namespace OrchardCore.Media.Core.GraphQL
```

Comment:
Let's not change the namespace to avoid users having to fix their code if they are using this class